plugins/ingest-attachment/src/test/resources/rest-api-spec/test/ingest_attachment/10_basic.yaml

```diff
@@ -8,4 +8,19 @@
         nodes.info: {}
 
     - match:  { nodes.$master.plugins.0.name: ingest-attachment  }
-    - match:  { nodes.$master.ingest.processors.11.type: attachment }
+    - match:  { nodes.$master.ingest.processors.0.type: append }
```

Comment:
this api lists all available processor we can't keep checks like this in the general rest tests since they would fail if you run them in the clients since the clients run against a es node with modules which has grok but grok is not part of the integ_test_zip task which has no modules. also the bwc stuff runs against a dist with modules which would cause failures there too.